### ISSUE TITLE
Add support for image, video and audio DIP creation

### ISSUE THREAD

**Issue by luis100** [keeps/roda#673]
Add the ability of current plugin tools to create DIPs:
- [x] ImageMagick plugin to create DIPs for images (e.g. JPEG)
- [x] FFMpeg plugin to create DIPs for video (into HTML5 compatible format)
- [x] SoX plugin to create DIPs for audio (into HTML5 compatible format)

**Comment by luis100** [comment]
Missing 

- [ ] Default parameters for converting into the HTML5 players compatible format
- [ ] Add "dissemination" category
- [ ] Clarify "Command arguments" and "Output command arguments"

**Comment by luis100** [comment]
```
File format has Representation information.
```
```
File format doesn't have Representation information.
```

Report information is too cryptic, not clear which representation information exists and where it is. Change to:
```
File format is described in format "Extensible Markup Language (XML) 1.0" (dd1e605a-fad6-422f-896c-85ca4057d0c8). Format description has MIME type and PRONOM but not the format designation.
```
Note: name and id are fetched from the format description hit.

```
Could not find a format description with MIME type "application/xml" or PRONOM "fmt/110" or format designation "Extensible Markup Language".
```
Note: it can be OR or AND depending on the options.

**Comment by nunovieira220** [comment]
Last one solved on 90d5778eb6331d38440e136f72c985b80f377fe6
